Assets/Scripts/UI/TabbedMenu.cs

```diff
@@ -214,7 +214,7 @@ private void OnEnable()
         private void Start()
         {
             PlayerController.Instance.PlayerControls.BasicControls.OpenSchedule.performed += OpenScheduleOnPerformed;
-            _timer = TimerManager.Instance.GetTimer("LoopTimer");
+            //_timer = TimerManager.Instance.GetTimer("LoopTimer");
```

Comment:
You could probably remove this line if it's not going to be used to make the code a little cleaner.